src/FixedDataTableCellGroup.js

```diff
@@ -65,74 +69,155 @@ class FixedDataTableCellGroupImpl extends React.Component {
 
   componentWillMount() {
     this._initialRender = true;
+    this._staticCellArray = [];
   }
 
   componentDidMount() {
     this._initialRender = false;
   }
 
   render() /*object*/ {
-    var props = this.props;
-    var columns = props.columns;
-    var cells = new Array(columns.length);
-    var contentWidth = sumPropWidths(columns);
+    const props = this.props;
+    const columns = props.columns;
+    const columnGroupWidth = sumPropWidths(columns);
```

Comment:
It's confusing to call this a column group, since we have a column group feature, but this actually refers to a group of cells (e.g. fixed, scrollable, or fixed right).

Should we rename this cellGroupWidth to avoid that confusion and be consistent w/ other parts of the codebase?

Reply:
> It's confusing to call this a column group, since we have a column group feature, but this actually refers to a group of cells (e.g. fixed, scrollable, or fixed right).
> 
> Should we rename this cellGroupWidth to avoid that confusion and be consistent w/ other parts of the codebase?

sounds good